fission/src/ui/modals/configuring/SettingsModal.tsx

```diff
@@ -66,7 +66,7 @@ const SettingsModal: React.FC<ModalPropsImpl> = ({ modalId }) => {
                 saveSettings()
             }}
         >
-            <div className="flex overflow-y-auto flex-col gap-2 bg-background-secondary rounded-md p-2 min-w-[22vw]">
+            <div className="flex overflow-y-auto flex-col gap-2 bg-background-secondary rounded-md p-2 max-h-[60vh]">
```

Comment:
This looks like the same change from #1100.

Reply:
@BrandonPacewic I probably thought that I broke it in this PR then forgot about that until I did the hotfix. It's such a small fix so should I just close the hotfix pr?

Reply:
Either way, I would probably just revert the line change in this PR. Makes it more explicit for people to test but it's up to you.